### PR TITLE
Feat/iri widget enhancement

### PR DESCRIPTION
**Pull request by Pooya-Oladazimi**
Related to https://github.com/ts4nfdi/terminology-service-suite/issues/84

Added Features:
- Make the external link icon optional. Default True.
- Added a copy to the clipboard button for the iri link. Default False.
- Added URL prefix feature for iri. When given, the encoded iri gets appended to the prefix. 
- Added stories for the above features.

### CODE REVIEW

File: src/app/types.ts [hunk 1]
```diff
@@ -221,6 +221,21 @@ export type IriWidgetProps = ForcedIriObj & {
      * Color of the text, names, hex or rgb
      */
     color?: EuiLinkColor | string;
+
+    /**
+     * Indicates that the target is exteranl and needs an icon.
```
Comment (jusa3) [inline src/app/types.ts:226]: spelling mistake

**Review by jusa3 — CHANGES_REQUESTED** [review]
Nice new features! Please also add stories to the HTML Storybook, by exporting them in the IriWdgetHTML.stories.ts. I will adapt the readme and make it more clear for developers to update both, the React and html stories (#88)

And please merge the latest changes I made on the main branch, since this will activate the build check for pull requests. Thank you :)

**Comment by Pooya-Oladazimi** [comment]
Not ready yet. The new features do not work in the HTML version for some unknown reason. The stories are there but the input props have no effect.

**Review by jusa3 — APPROVED** [review]

File: src/app/types.ts [hunk 2]
```diff
@@ -221,6 +221,21 @@ export type IriWidgetProps = ForcedIriObj & {
      * Color of the text, names, hex or rgb
      */
     color?: EuiLinkColor | string;
+
+    /**
+     * Indicates that the target is external and needs an icon.
+     */
+    externalIcon?: string;
```
[the remark this answers is not in the document]
Comment (Pooya-Oladazimi) [reply src/app/types.ts:228]: Yes true. I can't remember why I set it to string. I will do a fix-commit for this.

**Review by jusa3 — APPROVED** [review]

**Pull request merged**